pytorch_lightning/trainer/deprecated_api.py

```diff
@@ -153,3 +153,19 @@ def get_model(self) -> LightningModule:
             " and will be removed in v1.4.", DeprecationWarning
         )
         return self.lightning_module
+
+    @property
+    def running_sanity_check(self) -> bool:
+        rank_zero_warn(
```

Comment:
canot see a test for this deprecation

Reply:
It's there https://github.com/PyTorchLightning/pytorch-lightning/blob/6c6752c65133a8b70cec1ff5165ba9b2686dea63/tests/deprecated_api/test_remove_1-5.py#L59-L62